src/HealthGPS.Console/command_options.cpp

```diff
@@ -61,17 +61,7 @@ CommandOptions parse_arguments(cxxopts::Options &options, int &argc, char *argv[
         }
 
         if (result.count("storage")) {
-            cmd.storage_folder = result["storage"].as<std::string>();
-            if (cmd.storage_folder.is_relative()) {
-                cmd.storage_folder = std::filesystem::absolute(cmd.storage_folder);
-                fmt::print("File storage folder.: {}\n", cmd.storage_folder.string());
-            }
-        }
-
-        if (!fs::exists(cmd.storage_folder)) {
-            fmt::print(fg(fmt::color::red), "\nFile storage folder: {} not found.\n",
-                       cmd.storage_folder.string());
-            cmd.exit_code = EXIT_FAILURE;
+            cmd.data_path_or_url = result["storage"].as<std::string>();
```

Comment:
I guess it doesn't make sense any more to use a relative path in the config file?

Reply:
No, it does! I must have just deleted this without looking too closely at it. Oops :see_no_evil: 

I'll fix it up.

Reply:
Oh wait. This is a bit weird. It converts relative paths to absolute ones, but it doesn't actually change what the path points to. I assumed it was treating relative paths as if they were relative to the config file's path (and then fixing them up), but they're actually just relative to the CWD.

I suppose either way, if users are expecting to see absolute paths in log files etc., we should probably keep this behaviour though. What do you think?

If we were writing this from scratch, I think a better behaviour would be for paths to be relative to the config file, but I guess we'll break things if we change it now.

Reply:
> I suppose either way, if users are expecting to see absolute paths in log files etc., we should probably keep this behaviour though. What do you think?

Yeah, probably for the best. Rather than fiddling around with data dir location and config relative paths -- sounds error prone.

Reply:
It's easy enough to do, but I don't want to break people's existing config files.
Hopefully going forward everyone will use URLs instead anyway!